Tests/TuistGeneratorTests/Generator/LinkGeneratorTests.swift

```diff
@@ -482,4 +515,19 @@ final class LinkGeneratorErrorTests: XCTestCase {
 
         return projectFileElements
     }
+    
+    private func createFileElements(fileAbsolutePath: AbsolutePath,
```

Comment:
awesome! thanks for making this helper 👍 

One last suggestion to simplify it even further, we can prime the `ProjectFileElements` directly with the fileElement.

```swift
 private func createFileElements(fileAbsolutePath: AbsolutePath) -> ProjectFileElements {
        let fileElements = ProjectFileElements()
        fileElements.elements[fileAbsolutePath] = PBXFileReference(path: fileAbsolutePath.basename)
        return fileElements
    }
```

Something like this makes it an even more re-usable helper here as it not specific to the `Test.xcframework`